tests/test_listings.py

```diff
@@ -0,0 +1,100 @@
+"""
+Unit tests for listing methods to have expected columns
+with CashCtrlClient.
+"""
+
+import pandas as pd
+from cashctrl_api import CashCtrlClient, constants
+
+# Test function with assertions to check the tax rates DataFrame
+def test_list_tax_rates_to_have_expected_columns():
+    """
+    Test that the DataFrame returned by CashCtrlClient contains all expected columns
+    with the correct data types as specified in TAX_COLUMNS.
+    """
+    # Given: CashCtrlClient returns a DataFrame of tax rates
+    cc_client = CashCtrlClient()
+    tax_rates = cc_client.list_tax_rates()
+
+    assert isinstance(tax_rates, pd.DataFrame), "`tax_rates` is not a DataFrame."
+
+    # Check if all expected columns are in the DataFrame
+    assert set(constants.TAX_COLUMNS.keys()).issubset(
+        tax_rates.columns
+    ), "Some expected columns are missing."
+
+    # Check for unexpected columns in the DataFrame
+    unexpected_columns = set(tax_rates.columns) - set(constants.TAX_COLUMNS.keys())
+    assert not unexpected_columns, (
+        f"Unexpected columns found: {unexpected_columns}."
+    )
```

Comment:
The two tests "Check if all expected columns are present" and "Ensure no unexpected columns" can be simplified as a single test for set equality. (Applies to all list types)

Example:
```
a = {1, 2, 6, 7, 8}
b = {8, 6, 7, 2, 1}

# The two tests in the code
assert b.issubset(a), "Some expected columns are missing."
assert not (a - b),  "Unexpected columns found"

# Are equivalent to
assert a == b, "Columns are not identical"
```

Reply:
Thanks, this is now implemented for `test_list_accounts_to_have_expected_columns`.

Please also implement for taxes, accounts, and journal.

Btw: I would personally drop the comment, it is redundant with the error message.